### ISSUE TITLE
"ros setup" not reporting installation failures/fails silently on Ubuntu if ASDF already installed

### ISSUE THREAD

**Issue by phlummox** [roswell/roswell#487]
tl;dr: `ros setup` is not properly reporting failures that occur during setup; the problem with ASDF is just a symptom of this.

## Environment

```
$ uname -a
Linux breaker 5.4.0-70-generic #78~18.04.1-Ubuntu SMP Sat Mar 20 14:10:07 UTC 2021 x86_64 x86_64 x86_64 GNU/Linux
$ ros --version
roswell 21.06.14.110(NO-GIT-REVISION)
$ ros config
setup.time=3834929155
sbcl-bin.version=2.1.6
default.lisp=sbcl-bin

Possible subcommands:
set
show
```

## How to reproduce the issue
 
On Ubuntu 18.04, it can be reproduced by:

1\. Installing `ros` (e.g. from [release v21.06.14.110][rel-110], by running `sudo make install`), but not yet doing `ros setup`. 

[rel-110]: https://github.com/roswell/roswell/releases/download/v21.06.14.110/roswell-21.06.14.110-linux-x86_64.tar.bz2

2\. Then running:

```sh
$ sudo apt-get install cl-asdf
$ sudo ros setup
```

## Expected behavior

Either the invocation of `ros setup` should succeed fully (including correct installation of Quicklisp), or it should fail with an informative error message.

```sh
$ ros setup
Installing sbcl-bin...
No SBCL version specified. Downloading sbcl-bin_uri.tsv to see the available versions...
[##########################################################################]100%
Installing sbcl-bin/2.1.6...
Downloading https://github.com/roswell/sbcl_bin/releases/download/2.1.6/sbcl-2.1.6-x86-64-linux-binary.tar.bz2
[##########################################################################]100%
Extracting sbcl-bin-2.1.6-x86-64-linux.tar.bz2 to /home/phlummox/.roswell/src/sbcl-2.1.6-x86-64-linux/
Building sbcl-bin/2.1.6... Done.
Install Script for sbcl-bin...
Installing Quicklisp... Done 24747
Making core for Roswell...
$ ls ~/.roswell/lisp/quicklisp/dists/quicklisp/archives
access-20210124-git.tgz                 documentation-utils-20190710-git.tgz
alexandria-20210411-git.tgz             flexi-streams-20200925-git.tgz
anaphora-20210124-git.tgz               gettext-20171130-git.tgz
arnesi-20170403-git.tgz                 iterate-release-b0f9a9c6-git.tgz
array-utils-20201220-git.tgz            let-plus-20191130-git.tgz
babel-20200925-git.tgz                  local-time-20210124-git.tgz
cl-annot-20150608-git.tgz               named-readtables-20210531-git.tgz
cl-interpol-20201220-git.tgz            plump-20210630-git.tgz
cl-locale-20151031-git.tgz              ql-checkout-20190521-git.tgz
closer-mop-20210630-git.tgz             simple-date-time-20160421-git.tgz
cl-parser-combinators-20131111-git.tgz  split-sequence-v2.0.1.tgz
cl-ppcre-20190521-git.tgz               symbol-munger-20150407-git.tgz
cl-slice-20210531-git.tgz               trivial-backtrace-20200610-git.tgz
cl-syntax-20150407-git.tgz              trivial-features-20210411-git.tgz
cl-unicode-20210228-git.tgz             trivial-gray-streams-20210124-git.tgz
cl-yacc-20101006-darcs.tgz              trivial-indent-20210531-git.tgz
collectors-20161204-git.tgz             trivial-types-20120407-git.tgz
djula-20210630-git.tgz
```

## Actual result

`ros setup` apparently attempts to install QuickLisp three times, and fails silently – no error message is given indicating that installation did not actually succeed. 

```sh
$ ros setup
Installing sbcl-bin...
No SBCL version specified. Downloading sbcl-bin_uri.tsv to see the available versions...
[##########################################################################]100%
Installing sbcl-bin/2.1.6...
Downloading https://github.com/roswell/sbcl_bin/releases/download/2.1.6/sbcl-2.1.6-x86-64-linux-binary.tar.bz2
[##########################################################################]100%
Extracting sbcl-bin-2.1.6-x86-64-linux.tar.bz2 to /home/phlummox/.roswell/src/sbcl-2.1.6-x86-64-linux/
Building sbcl-bin/2.1.6... Done.
Install Script for sbcl-bin...
Installing Quicklisp... Done 13176
building dump:/home/phlummox/.roswell/env/roswell/impls/x86-64/linux/sbcl-bin/2.1.6/dump/roswell.core
Making core for Roswell...
Installing Quicklisp... Done 10006
building dump:/home/phlummox/.roswell/env/roswell/impls/x86-64/linux/sbcl-bin/2.1.6/dump/roswell.core
Installing Quicklisp... Done 10006
building dump:/home/phlummox/.roswell/env/roswell/impls/x86-64/linux/sbcl-bin/2.1.6/dump/roswell.core
$ ls ~/.roswell/lisp/quicklisp/dists/quicklisp/archives
ls: cannot access '/home/phlummox/.roswell/lisp/quicklisp/dists/quicklisp/archives': No such file or directory
$ ls ~/.roswell/lisp/quicklisp/
local-init
```

## Comments

On Ubuntu, ASDF (package `cl-asdf`) is a prerequisite of `slime`, so if a user has ever installed `slime`, they may well have it on their system (and possibly not know it). It gets installed into `/usr/share/common-lisp/source/cl-asdf`.

The following `Dockerfile` can be used to easily reproduce the issue:

```
FROM ubuntu:bionic

RUN apt-get clean && \
  apt-get update && \
  apt-get install -y  --no-install-recommends \
    bzip2       \
    ca-certificates \
    curl        \
    git         \
    make        \
    tar         \
    wget        \
    zlib1g-dev

RUN mkdir /work
WORKDIR /work

# install ros
RUN \
  curl -L https://github.com/roswell/roswell/releases/download/v21.06.14.110/roswell-21.06.14.110-linux-x86_64.tar.bz2 | tar xf - --bzip2 --strip-components=1 && \
  make install
```

If this is built and then run, it can be confirmed that `ros setup` works in the container correctly and QuickLisp is installed.  However, if, before running `ros setup`, one runs `sudo apt-get install -y --no-install-recommends cl-asdf`, then QuickLisp will not be installed correctly.

Obviously, if users remove the `cl-asdf` package[1], then QuickLisp will be able to be installed. However, this does not fix the actual bug: namely, that `ros setup` doesn't check the results of the QuickLisp installation to see if it succeeded (or, if it does check, does not report the failure to users in an error message).

[1] It might be worth adding a field to the `.deb` package saying "`conflicts: cl-asdf`", which would prevent a user from having both installed at the same time.

**Comment by phlummox** [comment]
On a positive note, I suspect removing the `cl-asdf` package on Ubuntu probably solves issue #454, and perhaps other issues that have been reported on Ubuntu.

**Comment by phlummox** [comment]
Okay. I managed to track down in which program (out of the many that are spawned during the `ros setup` process) the error seems to be occurring, by running `strace -v -f ros setup`.

It looks like, internally, `ros setup` spawns a process to run `ros install quicklisp`. According to the `strace` logs, this second process later makes an `execve` system call that is equivalent to running the following from the shell prompt:

```
$ SBCL_HOME=/root/.roswell/impls/x86-64/linux/sbcl-bin/2.1.6/lib/sbcl \
    INSTALL_ROOT=/root/.roswell/impls/x86-64/linux/sbcl-bin/2.1.6   \
    ROS_OPTS="((\"program\"\"(:eval\\\"(ros:quicklisp)\\\")\")(\"uname-m\"\"x86-64\")(\"uname\"\"linux\")(\"patchdir\"\"/usr/local/etc/roswell/patch/\")(\"lispdir\"\"/usr/local/etc/roswell/\")(\"verbose\"\"0\")(\"homedir\"\"/root/.roswell/\")(\"wargv0\"\"/usr/local/bin/ros\")(\"argv0\"\"/usr/local/bin/ros\")(\"quicklisp\"\"/root/.roswell/lisp/quicklisp/\")(\"impl\"\"sbcl-bin/2.1.6\")(\"script\"\"\\\"/usr/local/etc/roswell/install.ros\\\"\\\"quicklisp\\\"\")(\"lisp\"\"sbcl-bin\")(\"setup.time\"\"3834909923\")(\"sbcl-bin.version\"\"2.1.6\")(\"default.lisp\"\"sbcl-bin\"))"  \
    ROSINSTALL=sbcl-bin,NIL     \
    /root/.roswell/impls/x86-64/linux/sbcl-bin/2.1.6/bin/sbcl \
      --core /root/.roswell/impls/x86-64/linux/sbcl-bin/2.1.6/lib/sbcl/sbcl.core  \
      --noinform --no-sysinit  --no-userinit --disable-debugger \
      --eval "(progn #-ros.init(cl:load \"/usr/local/etc/roswell/init.lisp\"))" \
      --eval "(ros:run '((:eval\"(ros:quicklisp)\")(:script \"/usr/local/etc/roswell/install.ros\"\"quicklisp\")(:quit ())))"
```

If one runs the above shell command while `cl-asdf` is installed (I ran it in the Docker container shown in the #487 issue report), the following output can be seen:

```
Skip downloading http://beta.quicklisp.org/quicklisp.lisp.
Specify 'download.force=t' to download again.
; 
; caught WARNING:
;   :SB-EVAL is no longer present in *FEATURES*
ASDF could not load sb-bsd-sockets because
COMPILE-FILE-ERROR while compiling #<CL-SOURCE-FILE "asdf" "build" "asdf">.
While evaluating the form starting at line 314, column 0
  of #P"/root/.roswell/archives/quicklisp.lisp":
Unhandled UIOP/LISP-BUILD:COMPILE-FILE-ERROR in thread #<SB-THREAD:THREAD "main thread" RUNNING
                                                          {1001550E33}>:
  COMPILE-FILE-ERROR while compiling #<CL-SOURCE-FILE "asdf" "build" "asdf">

Backtrace for: #<SB-THREAD:THREAD "main thread" RUNNING {1001550E33}>
0: (SB-DEBUG::DEBUGGER-DISABLED-HOOK #<UIOP/LISP-BUILD:COMPILE-FILE-ERROR {10043811A3}> #<unused argument> :QUIT T)
1: (SB-DEBUG::RUN-HOOK SB-EXT:*INVOKE-DEBUGGER-HOOK* #<UIOP/LISP-BUILD:COMPILE-FILE-ERROR {10043811A3}>)
2: (INVOKE-DEBUGGER #<UIOP/LISP-BUILD:COMPILE-FILE-ERROR {10043811A3}>)
3: (ERROR UIOP/LISP-BUILD:COMPILE-FILE-ERROR :CONTEXT-FORMAT "~/asdf-action::format-action/" :CONTEXT-ARGUMENTS ((#<ASDF/LISP-ACTION:COMPILE-OP > . #<ASDF/LISP-ACTION:CL-SOURCE-FILE "asdf" "build" "asdf">)))
4: (UIOP/LISP-BUILD:CHECK-LISP-COMPILE-RESULTS NIL T T "~/asdf-action::format-action/" ((#<ASDF/LISP-ACTION:COMPILE-OP > . #<ASDF/LISP-ACTION:CL-SOURCE-FILE "asdf" "build" "asdf">)))
5: ((SB-PCL::EMF ASDF/ACTION:PERFORM) #<unused argument> #<unused argument> #<ASDF/LISP-ACTION:COMPILE-OP > #<ASDF/LISP-ACTION:CL-SOURCE-FILE "asdf" "build" "asdf">)
6: ((LAMBDA NIL :IN ASDF/ACTION:CALL-WHILE-VISITING-ACTION))
7: ((:METHOD ASDF/ACTION:PERFORM-WITH-RESTARTS :AROUND (T T)) #<ASDF/LISP-ACTION:COMPILE-OP > #<ASDF/LISP-ACTION:CL-SOURCE-FILE "asdf" "build" "asdf">) [fast-method]
8: ((:METHOD ASDF/PLAN:PERFORM-PLAN (T)) #<ASDF/PLAN:SEQUENTIAL-PLAN {1004127233}>) [fast-method]
9: ((FLET SB-C::WITH-IT :IN SB-C::%WITH-COMPILATION-UNIT))
10: ((:METHOD ASDF/PLAN:PERFORM-PLAN :AROUND (T)) #<ASDF/PLAN:SEQUENTIAL-PLAN {1004127233}>) [fast-method]
11: ((:METHOD ASDF/OPERATE:OPERATE (ASDF/OPERATION:OPERATION ASDF/COMPONENT:COMPONENT)) #<ASDF/LISP-ACTION:LOAD-OP > #<ASDF/PACKAGE-INFERRED-SYSTEM:PACKAGE-INFERRED-SYSTEM "asdf"> :PLAN-CLASS NIL :PLAN-OPTIONS NIL) [fast-method]
12: ((SB-PCL::EMF ASDF/OPERATE:OPERATE) #<unused argument> #<unused argument> #<ASDF/LISP-ACTION:LOAD-OP > #<ASDF/PACKAGE-INFERRED-SYSTEM:PACKAGE-INFERRED-SYSTEM "asdf"> :VERBOSE NIL)
13: ((LAMBDA NIL :IN ASDF/OPERATE:OPERATE))
14: ((:METHOD ASDF/OPERATE:OPERATE :AROUND (T T)) #<ASDF/LISP-ACTION:LOAD-OP > #<ASDF/PACKAGE-INFERRED-SYSTEM:PACKAGE-INFERRED-SYSTEM "asdf"> :VERBOSE NIL) [fast-method]
15: ((SB-PCL::EMF ASDF/OPERATE:OPERATE) #<unused argument> #<unused argument> ASDF/LISP-ACTION:LOAD-OP :ASDF :VERBOSE NIL)
16: ((LAMBDA NIL :IN ASDF/OPERATE:OPERATE))
17: ((:METHOD ASDF/OPERATE:OPERATE :AROUND (T T)) ASDF/LISP-ACTION:LOAD-OP :ASDF :VERBOSE NIL) [fast-method]
18: (ASDF/OPERATE:LOAD-SYSTEM :ASDF :VERBOSE NIL)
19: (ASDF/UPGRADE:UPGRADE-ASDF)
20: ((LAMBDA NIL :IN ASDF/OPERATE:OPERATE))
21: ((:METHOD ASDF/OPERATE:OPERATE :AROUND (T T)) #<ASDF/FIND-SYSTEM:DEFINE-OP > #<ASDF/SYSTEM:UNDEFINED-SYSTEM "sb-bsd-sockets">) [fast-method]
22: ((LAMBDA NIL :IN ASDF/FIND-SYSTEM:LOAD-ASD))
23: ((LAMBDA NIL :IN ASDF/SYSTEM:FIND-SYSTEM))
24: (ASDF/SESSION:CONSULT-ASDF-CACHE (ASDF/SYSTEM:FIND-SYSTEM "sb-bsd-sockets") #<FUNCTION (LAMBDA NIL :IN ASDF/SYSTEM:FIND-SYSTEM) {1001910B3B}>)
25: (ASDF/SESSION:CALL-WITH-ASDF-SESSION #<FUNCTION (LAMBDA NIL :IN ASDF/SYSTEM:FIND-SYSTEM) {1001910B3B}> :OVERRIDE NIL :KEY (ASDF/SYSTEM:FIND-SYSTEM "sb-bsd-sockets") :OVERRIDE-CACHE NIL :OVERRIDE-FORCING NIL)
26: (ASDF/OPERATE:MODULE-PROVIDE-ASDF SB-BSD-SOCKETS)
27: ((FLET "WRAPPER4" :IN REQUIRE) ASDF/OPERATE:MODULE-PROVIDE-ASDF)
28: (SB-KERNEL:%MAP-FOR-EFFECT-ARITY-1 #<FUNCTION (FLET "WRAPPER4" :IN REQUIRE) {7F92409361AB}> (ASDF/OPERATE:MODULE-PROVIDE-ASDF SB-IMPL::MODULE-PROVIDE-CONTRIB))
29: (REQUIRE SB-BSD-SOCKETS NIL)
30: (SB-INT:SIMPLE-EVAL-IN-LEXENV (REQUIRE (QUOTE SB-BSD-SOCKETS)) #<NULL-LEXENV>)
31: (SB-IMPL::SIMPLE-EVAL-PROGN-BODY ((REQUIRE (QUOTE SB-BSD-SOCKETS)) (DEFCLASS SBCL NIL NIL) (DEFPACKAGE #:QLQS-SBCL (:EXPORT . #1=(#:GET-HOST-BY-NAME #:INET-SOCKET #:HOST-ENT-ADDRESS #:SOCKET-CONNECT #:SOCKET-MAKE-STREAM)) (:EXPORT . #2=(#:COMPILER-NOTE)) (:INTERN #:HOST-NETWORK-ADDRESS) (:DOCUMENTATION "Steel Bank Common Lisp - http://www.sbcl.org/") (:USE) (:EXPORT #:LISP) (:IMPORT-FROM #:SB-BSD-SOCKETS . #1#) (:IMPORT-FROM #:SB-EXT . #2#)) (SETF *IMPLEMENTATION* (MAKE-INSTANCE (QUOTE SBCL)))) #<NULL-LEXENV>)
32: (SB-INT:SIMPLE-EVAL-IN-LEXENV (DEFINE-IMPLEMENTATION-PACKAGE :SBCL #:QLQS-SBCL (:CLASS SBCL) (:DOCUMENTATION "Steel Bank Common Lisp - http://www.sbcl.org/") (:PREP (REQUIRE (QUOTE SB-BSD-SOCKETS))) (:INTERN #:HOST-NETWORK-ADDRESS) (:REEXPORT-FROM #:SB-EXT #:COMPILER-NOTE) (:REEXPORT-FROM #:SB-BSD-SOCKETS #:GET-HOST-BY-NAME #:INET-SOCKET #:HOST-ENT-ADDRESS #:SOCKET-CONNECT #:SOCKET-MAKE-STREAM)) #<NULL-LEXENV>)
33: (SB-EXT:EVAL-TLF (DEFINE-IMPLEMENTATION-PACKAGE :SBCL #:QLQS-SBCL (:CLASS SBCL) (:DOCUMENTATION "Steel Bank Common Lisp - http://www.sbcl.org/") (:PREP (REQUIRE (QUOTE SB-BSD-SOCKETS))) (:INTERN #:HOST-NETWORK-ADDRESS) (:REEXPORT-FROM #:SB-EXT #:COMPILER-NOTE) (:REEXPORT-FROM #:SB-BSD-SOCKETS #:GET-HOST-BY-NAME #:INET-SOCKET #:HOST-ENT-ADDRESS #:SOCKET-CONNECT #:SOCKET-MAKE-STREAM)) 31 NIL)
34: ((LABELS SB-FASL::EVAL-FORM :IN SB-INT:LOAD-AS-SOURCE) (DEFINE-IMPLEMENTATION-PACKAGE :SBCL #:QLQS-SBCL (:CLASS SBCL) (:DOCUMENTATION "Steel Bank Common Lisp - http://www.sbcl.org/") (:PREP (REQUIRE (QUOTE SB-BSD-SOCKETS))) (:INTERN #:HOST-NETWORK-ADDRESS) (:REEXPORT-FROM #:SB-EXT #:COMPILER-NOTE) (:REEXPORT-FROM #:SB-BSD-SOCKETS #:GET-HOST-BY-NAME #:INET-SOCKET #:HOST-ENT-ADDRESS #:SOCKET-CONNECT #:SOCKET-MAKE-STREAM)) 31)
35: ((LAMBDA (SB-KERNEL:FORM &KEY :CURRENT-INDEX &ALLOW-OTHER-KEYS) :IN SB-INT:LOAD-AS-SOURCE) (DEFINE-IMPLEMENTATION-PACKAGE :SBCL #:QLQS-SBCL (:CLASS SBCL) (:DOCUMENTATION "Steel Bank Common Lisp - http://www.sbcl.org/") (:PREP (REQUIRE (QUOTE SB-BSD-SOCKETS))) (:INTERN #:HOST-NETWORK-ADDRESS) (:REEXPORT-FROM #:SB-EXT #:COMPILER-NOTE) (:REEXPORT-FROM #:SB-BSD-SOCKETS #:GET-HOST-BY-NAME #:INET-SOCKET #:HOST-ENT-ADDRESS #:SOCKET-CONNECT #:SOCKET-MAKE-STREAM)) :CURRENT-INDEX 31)
36: (SB-C::%DO-FORMS-FROM-INFO #<FUNCTION (LAMBDA (SB-KERNEL:FORM &KEY :CURRENT-INDEX &ALLOW-OTHER-KEYS) :IN SB-INT:LOAD-AS-SOURCE) {10040B198B}> #<SB-C::SOURCE-INFO {10040B1953}> SB-C::INPUT-ERROR-IN-LOAD)
37: (SB-INT:LOAD-AS-SOURCE #<SB-INT:FORM-TRACKING-STREAM for "file /root/.roswell/archives/quicklisp.lisp" {10040B0393}> :VERBOSE NIL :PRINT NIL :CONTEXT "loading")
38: ((LABELS SB-FASL::LOAD-STREAM-1 :IN LOAD) #<SB-INT:FORM-TRACKING-STREAM for "file /root/.roswell/archives/quicklisp.lisp" {10040B0393}> NIL)
39: (SB-FASL::CALL-WITH-LOAD-BINDINGS #<FUNCTION (LABELS SB-FASL::LOAD-STREAM-1 :IN LOAD) {7F9240936BAB}> #<SB-INT:FORM-TRACKING-STREAM for "file /root/.roswell/archives/quicklisp.lisp" {10040B0393}> NIL #<SB-INT:FORM-TRACKING-STREAM for "file /root/.roswell/archives/quicklisp.lisp" {10040B0393}>)
40: (LOAD #P"/root/.roswell/archives/quicklisp.lisp" :VERBOSE NIL :PRINT NIL :IF-DOES-NOT-EXIST T :EXTERNAL-FORMAT :DEFAULT)
41: (ROSWELL.INSTALL.QUICKLISP::QUICKLISP-INSTALL (:TARGET "quicklisp" :VERSION NIL :VERSION-NOT-SPECIFIED NIL :ARGV NIL))
42: (ROSWELL.INSTALL:INSTALL-IMPL "quicklisp" NIL NIL (ROSWELL.INSTALL.QUICKLISP::QUICKLISP-ARGV-PARSE ROSWELL.INSTALL.QUICKLISP::QUICKLISP-DOWNLOAD ROSWELL.INSTALL.QUICKLISP::QUICKLISP-INSTALL))
43: (ROSWELL.INSTALL:INSTALL-IMPL-IF-PROBED "quicklisp" NIL #<unavailable argument>)
44: (ROSWELL.INSTALL:INSTALL NIL)
45: (SB-INT:SIMPLE-EVAL-IN-LEXENV (APPLY (QUOTE ROS.SCRIPT.INSTALL.3671936092::MAIN) ROSWELL:*ARGV*) #<NULL-LEXENV>)
46: (SB-INT:SIMPLE-EVAL-IN-LEXENV (ROSWELL:QUIT (APPLY (QUOTE ROS.SCRIPT.INSTALL.3671936092::MAIN) ROSWELL:*ARGV*)) #<NULL-LEXENV>)
47: (SB-EXT:EVAL-TLF (ROSWELL:QUIT (APPLY (QUOTE ROS.SCRIPT.INSTALL.3671936092::MAIN) ROSWELL:*ARGV*)) NIL NIL)
48: ((LABELS SB-FASL::EVAL-FORM :IN SB-INT:LOAD-AS-SOURCE) (ROSWELL:QUIT (APPLY (QUOTE ROS.SCRIPT.INSTALL.3671936092::MAIN) ROSWELL:*ARGV*)) NIL)
49: (SB-INT:LOAD-AS-SOURCE #<CONCATENATED-STREAM :STREAMS NIL {10030318A3}> :VERBOSE NIL :PRINT NIL :CONTEXT "loading")
50: ((LABELS SB-FASL::LOAD-STREAM-1 :IN LOAD) #<CONCATENATED-STREAM :STREAMS NIL {10030318A3}> NIL)
51: (SB-FASL::CALL-WITH-LOAD-BINDINGS #<FUNCTION (LABELS SB-FASL::LOAD-STREAM-1 :IN LOAD) {7F92409376CB}> #<CONCATENATED-STREAM :STREAMS NIL {10030318A3}> NIL #<CONCATENATED-STREAM :STREAMS NIL {10030318A3}>)
52: (LOAD #<CONCATENATED-STREAM :STREAMS NIL {10030318A3}> :VERBOSE NIL :PRINT NIL :IF-DOES-NOT-EXIST T :EXTERNAL-FORMAT :DEFAULT)
53: ((FLET ROSWELL::BODY :IN ROSWELL:SCRIPT) #<SB-SYS:FD-STREAM for "file /usr/local/etc/roswell/install.ros" {10030290A3}>)
54: (ROSWELL:SCRIPT "/usr/local/etc/roswell/install.ros" "quicklisp")
55: (ROSWELL:RUN ((:EVAL "(ros:quicklisp)") (:SCRIPT "/usr/local/etc/roswell/install.ros" "quicklisp") (:QUIT NIL)))
56: (SB-INT:SIMPLE-EVAL-IN-LEXENV (ROSWELL:RUN (QUOTE ((:EVAL "(ros:quicklisp)") (:SCRIPT "/usr/local/etc/roswell/install.ros" "quicklisp") (:QUIT NIL)))) #<NULL-LEXENV>)
57: (EVAL (ROSWELL:RUN (QUOTE ((:EVAL "(ros:quicklisp)") (:SCRIPT "/usr/local/etc/roswell/install.ros" "quicklisp") (:QUIT NIL)))))
58: (SB-IMPL::PROCESS-EVAL/LOAD-OPTIONS ((:EVAL . "(progn #-ros.init(cl:load \"/usr/local/etc/roswell/init.lisp\"))") (:EVAL . "(ros:run '((:eval\"(ros:quicklisp)\")(:script \"/usr/local/etc/roswell/install.ros\"\"quicklisp\")(:quit ())))")))
59: (SB-IMPL::TOPLEVEL-INIT)
60: ((FLET SB-UNIX::BODY :IN SB-IMPL::START-LISP))
61: ((FLET "WITHOUT-INTERRUPTS-BODY-1" :IN SB-IMPL::START-LISP))
62: (SB-IMPL::START-LISP)

unhandled condition in --disable-debugger mode, quitting
; 
; compilation unit aborted
;   caught 1 fatal ERROR condition
;   caught 1 WARNING condition
```

Now, I'm not sure if this is the "root cause" of the problem, or just a consequence of a problem that's already occurred, but either way, it's clearly a failure, and something about it should be reported to the end user.  Showing it to the Roswell end-user would at least give them a chance at diagnosing the problem; hiding it makes it very difficult for the user to know what went wrong.
